Multi_Agent_Robot/multi_agent_robot/env/agent_action_space.py

```diff
@@ -0,0 +1,22 @@
+from gym.spaces import Discrete
+
+
+class AgentActionSpace(Discrete):
```

Comment:
maybe make the oracle's action space a different class? also we might need to sync if we are going to work on the oracle together